source/_integrations/surepetcare.markdown

```diff
@@ -55,6 +55,21 @@ surepetcare:
     type: time
 {% endconfiguration %}
 
+## set_lock_state Service
+
+This service let you change the locking state of a flap
+
+| Service data attribute | Required | Type | Description |
+| ---------------------- | -------- | -------- | ----------- |
+| `flap_id` | `True` | integer | Flap ID to change - see below for instructions on finding device IDs
```

Comment:
Why isn't this tied to the entity_ids the integration provides?

Reply:
Yeah, I was unsure about the best way to approach this. There isn't really a natural entity to target - the closest we've got is the sensor that reflects the current lock state. It feels a bit weird/unintuitive to me to need to specify a sensor entity, when what I really want to act on is the flap itself. Requiring a user to know the flap ID isn't an especially bad UX, they already need to know it to configure the integration.

If this goes against established patterns, I'll happily rework the PR to fit.